experimental/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -36,6 +36,10 @@ export interface TinyliciousResources {
     containerServices: TinyliciousContainerServices;
 }
 
+export interface TinyliciousDetachedResources extends TinyliciousResources {
```

Comment:
I really don't want to use the detached terminology.

```suggestion
export interface TinyliciousCreateResources extends TinyliciousResources {
```